frontend/src/pages/ProjectSettings/ErrorFiltersForm/ErrorFiltersForm.tsx

```diff
@@ -0,0 +1,99 @@
+import { FieldsBox } from '@components/FieldsBox/FieldsBox'
+import { CircularSpinner, LoadingBar } from '@components/Loading/Loading'
+import Select from '@components/Select/Select'
+import { useEditProjectMutation, useGetProjectQuery } from '@graph/hooks'
+import { namedOperations } from '@graph/operations'
+import { useParams } from '@util/react-router/useParams'
+import { message } from 'antd'
+import clsx from 'clsx'
+import React, { useEffect, useState } from 'react'
+
+import commonStyles from '../../../Common.module.scss'
+import Button from '../../../components/Button/Button/Button'
+import styles from './ErrorFiltersForm.module.scss'
+
+export const ErrorFiltersForm = () => {
+	const { project_id } = useParams<{
+		project_id: string
+	}>()
+	const [errorFilters, setErrorFilters] = useState<string[]>([])
+	const { data, loading } = useGetProjectQuery({
+		variables: {
+			id: project_id!,
+		},
+		skip: !project_id,
+	})
+
+	const [editProject, { loading: editProjectLoading }] =
+		useEditProjectMutation({
+			refetchQueries: [
+				namedOperations.Query.GetProjects,
+				namedOperations.Query.GetProject,
+			],
+		})
+
+	const onSubmit = (e: { preventDefault: () => void }) => {
+		e.preventDefault()
+		if (!project_id) {
+			return
+		}
+		editProject({
+			variables: {
+				id: project_id!,
+				error_filters: errorFilters,
+			},
+		}).then(() => {
+			message.success('Updated error filters!', 5)
+		})
+	}
+
+	useEffect(() => {
+		if (!loading) {
+			setErrorFilters(data?.project?.error_filters || [])
+		}
+	}, [data?.project?.error_filters, loading])
+
+	if (loading) {
+		return <LoadingBar />
+	}
+
+	return (
+		<FieldsBox id="errors">
```

Comment:
I've never seen this component but looking the code...it looks like it scrolls that part of the page into view if the URL hash matches:

https://github.com/highlight/highlight/blob/fae0d6edd3f0590bbe72b38a4b6a8d024e8d64cd/frontend/src/components/FieldsBox/FieldsBox.tsx#L13-L19

Yet another part of the page uses the same `id`:
https://github.com/highlight/highlight/blob/fae0d6edd3f0590bbe72b38a4b6a8d024e8d64cd/frontend/src/pages/ProjectSettings/ErrorSettingsForm/ErrorSettingsForm.tsx#L61

So these two are going to conflict with each other.

Honestly, I can't see a reason for us to actually use this component. The page isn't even that long so it feels unnecessary. Can we just omit the usage?

Reply:
updating the ids here for consistency, but the fields box sets styles and allows us to link directly to a section here (eg https://localhost:3000/2/settings/errors#sourcemaps)
<img width="725" alt="Screenshot 2023-04-09 at 4 24 53 PM" src="https://user-images.githubusercontent.com/1351531/230801289-53246164-156c-432d-951d-6d0725905814.png">